estela-web/README.md

```diff
@@ -1,6 +1,36 @@
-<h1 align="center">estela Web</h1>
+<h1 align="center">
+    <span>
+        <img width="38" style="vertical-align: middle" src="https://cdn-icons-png.flaticon.com/512/1078/1078191.png">
+    </span>
+    estela Web
+</h1>
 
-This project is created using React Framework (along with Ant Design) and Typescript. This module implements a
-user-friendly environment that communicates with the estela API and lets you manage your spiders and scraping
-projects. For more information on this module and how to set it up, please refer to our
-[official documentation](https://bitmaker.la/docs/bitmaker-cloud/web.html).
+![version](https://img.shields.io/badge/version-0.1-blue)
+![yarn-version](https://img.shields.io/badge/yarn-v1.22.19-blue)
+![build](https://img.shields.io/badge/build-passing-brightgreen)
+![CI](https://github.com/eslint/eslint/workflows/CI/badge.svg)
```

Comment:
I think we can get rid of these badges. There is a specific ticket to define this stuff for the whole project.